Moli/Core/Screens/Profile/ProfileStatsView.swift

```diff
@@ -0,0 +1,37 @@
+import SwiftUI
+
+struct ProfileStatsView: View {
+    var userInfo: UserAPI.UserResponse
+    var followers: String
+    var friends: String
+    var following: String
+
+    var statsDivider: some View {
+        HStack {Divider()}.frame(height: 44).padding(.horizontal)
+    }
+
+    var body: some View {
+        HStack {
```

Comment:
:art: Har insett att HStack:n inte ger spacing effekten till Dividers, testade koden fram och tillbaks, och kom fram med att slutresultatet ska se ut något som följande om den ska matcha design layout:n

 ```suggestion
    var statsDivider: some View {
        Divider()
        .padding(.horizontal)
    }

    var body: some View {
        HStack(spacing: 0) {
```